lib/eslint.js

```diff
@@ -599,7 +599,7 @@ module.exports = (function() {
      * @param {string} text The text to parse.
      * @param {Object} config The ESLint configuration object.
      * @param {string} filePath The path to the file being parsed.
-     * @returns {ASTNode} The AST if successful or null if not.
+     * @returns {ASTNode|Object} The AST if successful or null if not.
```

Comment:
Is the "Object" type (object with "ast" and "services" properties) worth documenting as a typedef?